IntegrationAdapters/Repositories/TenderRepository.cs

```diff
@@ -69,8 +69,26 @@ public List<TenderOffer> GetAllOffersByEmailAndTender(string emailAndTender)
                     foundOffers.Add(offer);
             }
             return foundOffers;
+        }
+        public bool IsOfferExisting(TenderOffer tenderOffer)
+        {
+            List<TenderOffer> tenderOffers = dbContext.TenderOffer.ToList();
+            foreach (TenderOffer tenderoffer in tenderOffers)
+            {
+                if (tenderoffer.Id.Equals(tenderOffer.Id)) return true;
+            }
+            return false;

```

Comment:
![image](https://user-images.githubusercontent.com/57258204/104179263-da5f8a80-540b-11eb-8f08-f968f830f3b0.png)
You can try something like this to enhance performance, dbContext.TenderOffers.Any(tenderoffer => tenderoffer.Id.Equals(tenderOffer.Id))